app/Theme/Plugins.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Dxw\MyTheme\Theme;
+
+use Dxw\Iguana\Registerable;
+
+class Plugins implements Registerable
+{
+    protected $required;
+    protected $abspath;
+
+    public function __construct(array $required = [], $abspath = null)
+    {
+        $this->required = $required;
+        $this->abspath = $abspath ?? ABSPATH;
+    }
+
+    public function register()
+    {
+        add_action('after_switch_theme', [$this, 'checkDependencies']);
```

Comment:
Is this sufficient? Are there any other ways in which a theme can become activated?